### ISSUE TITLE
print output to text format

### ISSUE THREAD

**Issue by nouri-muhammad** [SLU-CS-Capstone/maze_generator#6]
Hi, I'm beginner to programming and I liked this project. 
I can add the functions so the program prints output as text format as well as displaying the result in terminal in the same directory as generator file. It is possible to ask user as to which output they prefer. 
is that OK if I do it?

**Comment by Stuartwastaken** [comment]
Yep that sounds great. We would love for you to join the project! Thanks for showing your interest. Just make a new github issue to address that. :)
